src/e2e/java/teammates/e2e/cases/e2e/BaseE2ETestCase.java

```diff
@@ -50,6 +55,7 @@ protected String getTestDataFolder() {
     @AfterClass(alwaysRun = true)
     public void baseClassTearDown() {
         releaseBrowser();
+        removeTestData();
```

Comment:
Why you need to removeTestData? Is it done in `removeAndRestoreDataBundle`?

Reply:
I thought it would be better to remove test data after running the tests instead of letting it remain in the datastore. 

Reverted the change